### ISSUE TITLE
[CEM] Custom parrot model doesn't apply to shoulder parrots

### ISSUE THREAD

**Issue by DragendGhast** [sp614x/optifine#2295]
Parrot CEM reverts to original model when on a player's shoulder.

**Comment by realjackasterisk** [comment]
duplicate of #2179 but at least this ones in plain english.

**Comment by sp614x** [comment]
It was fixed for 1.12.2 in #967.
Still open for 1.13.2.

**Comment by sp614x** [comment]
Still not working in 1.14.4.

**Comment by BlevinsTyler** [comment]
not working in 1.16.2

**Comment by ColonelOverspel** [comment]
Not working in 1.16.4!

**Comment by ZenithRogue** [comment]
Issue still present in 1.17.1

**Comment by sp614x** [comment]
Fixed, coming in next preview.
